pkgs/servers/cloud9/default.nix

```diff
@@ -0,0 +1,93 @@
+{ stdenv ? (import <nixpkgs> {}).stdenv
+, lib ? (import <nixpkgs> {}).pkgs.lib
+, pkgs ? (import <nixpkgs> {}).pkgs
+, nodePackages ? (import <nixpkgs> {}).pkgs.nodePackages
+}:
+let
+inherit nodePackages;
+inherit lib;
+inherit pkgs;
```

Comment:
Thanks, I can lint after everything is done. (defaults are convenient
during testing). I'm interested in adopting buildNodePackage instead of the
adhoc mkDerivation it uses now.
On Apr 4, 2016 3:27 AM, "joachifm" <notifications@github.com> wrote:

> In pkgs/servers/cloud9/default.nix
> <https://github.com/NixOS/nixpkgs/pull/14437#discussion_r58336991>:
>
> > @@ -0,0 +1,93 @@
> > +{ stdenv ? (import <nixpkgs> {}).stdenv
> > +, lib ? (import <nixpkgs> {}).pkgs.lib
> > +, pkgs ? (import <nixpkgs> {}).pkgs
> > +, nodePackages ? (import <nixpkgs> {}).pkgs.nodePackages
> > +}:
> > +let
> > +inherit nodePackages;
> > +inherit lib;
> > +inherit pkgs;
>
> These bindings will already be in scope so this accomplishes nothing.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/14437/files/6c76578fd83dbb47913dac4e4541236abbb8d393#r58336991>
>